### PR TITLE
[#47 #48] moved activities to activityPage and added new field for sign up

### PR DESCRIPTION
**Pull request by darkheros12**

### CODE REVIEW

**Review by Landry333 — CHANGES_REQUESTED** [review]
The GitHub actions checks failed. Can you fix that (only the GitHub actions checks as we will not be using Travis ci for now) ?

File: app/src/main/java/com/example/bigowlapp/ActivityPage/HomeActivity.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import com.example.bigowlapp.R;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class HomeActivity extends AppCompatActivity {
```
Comment (KhangZhengNg) [inline app/src/main/java/com/example/bigowlapp/ActivityPage/HomeActivity.java:13]: Good job on moving files into proper folder. However, please pull new code from Staging branch as an earlier PR has been approved

File: app/src/main/java/com/example/bigowlapp/ActivityPage/SignUpActivity.java [hunk 2]
```diff
@@ -57,38 +64,48 @@ else if(pass.isEmpty())
...
+                        phone.setError("please enter a phone number");
+                        phone.requestFocus();
                     }
-                    else if(!(email.isEmpty() && pass.isEmpty()))
+                    else if(email.isEmpty() || pass.isEmpty() || userPhone.isEmpty())
                     {
-                        m_FirebaseAuth.createUserWithEmailAndPassword(email, pass).addOnCompleteListener(MainActivity.this, new OnCompleteListener<AuthResult>() {
+                        Toast.makeText(SignUpActivity.this, "Fields are empty!", Toast.LENGTH_SHORT).show();
+                    }
+                    else if(!(email.isEmpty() && pass.isEmpty() &&  userPhone.isEmpty()))
```
Comment (KhangZhengNg) [inline app/src/main/java/com/example/bigowlapp/ActivityPage/SignUpActivity.java:76]: Good job on handling multiple possible inputs by a user.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -13,9 +13,9 @@
         <activity android:name=".ViewAnotherUserActivity"></activity>
         <activity android:name=".InvitationConfirmationActivity" />
         <activity android:name=".SendSmsInvitationActivity" />
-        <activity android:name=".HomeActivity" />
-        <activity android:name=".LoginActivity" />
-        <activity android:name=".MainActivity">
+        <activity android:name=".ActivityPage.HomeActivity" />
+        <activity android:name=".ActivityPage.LoginActivity" />
+        <activity android:name=".ActivityPage.SignUpActivity">
```
Comment (KhangZhengNg) [inline app/src/main/AndroidManifest.xml:18]: Good fix on these lines as the activity files have moved to a folder.

**Review by KhangZhengNg — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/bigowlapp/ActivityPage/HomeActivity.java (hunk 1 above)
Comment (darkheros12) [reply app/src/main/java/com/example/bigowlapp/ActivityPage/HomeActivity.java]: DONE

**Review by KhangZhengNg — APPROVED** [review]
Well done on fixing things! Now the new login, signup, and home pages looks awesone!

**Review by Landry333 — APPROVED** [review]
The requested changes have been done. It looks ok to me

**Pull request merged**